cukinia

```diff
@@ -349,6 +437,12 @@ _cukinia_mount()
 # arg2..: string
 _colorize()
 {
+	if [ -n "$__log_format" ] || [ "$__log_file" != "/dev/stdout" ]; then
+		local text=${*:2}
+		echo "${text}"
+		return
+	fi
+
 	local color="$1"; shift
```

Comment:
can we keep the local declarations at the top of the function ? what's the impact on the ${*:2} above?

Reply:
${*:2} is equivalent to a shift of 1 argument in $*, it takes the argument slice starting with the second one until the last; ${*:2:N} would take a slice of size N starting on the one. works with $* and $@.

but this will be useless with variable declaration at the top